extensions/oidc-client/runtime/src/main/java/io/quarkus/oidc/client/filter/OidcClientFilter.java

```diff
@@ -10,4 +10,11 @@
 @Retention(RetentionPolicy.RUNTIME)
 @Documented
 public @interface OidcClientFilter {
+
+    /**
+     * @return name of the OIDC client that should be used to acquire the tokens. Client names specified through
+     *         the annotation have higher priority than name specified through the `client-name` configuration property.
```

Comment:
```suggestion
     *         
```

Reply:
True, it's weird sentence, thanks. Btw. IMHO `registerFilter` can be easily done for reactive side and than we could merge them.